### PR TITLE
MWPW-143330: Blog-posts block

### PR DESCRIPTION
**Pull request by rl-adobe**
Import blog-posts block from express to milo.

Resolves: [MWPW-143330](https://jira.corp.adobe.com/browse/MWPW-143330)

Test URLs:
- Pre Migration Link: https://www.adobe.com/express/create/meme
- Before: https://main--express-milo--adobecom.hlx.page/express/create/meme
- After: https://rl-adobe-blog-posts--express-milo--adobecom.hlx.page/express/create/meme?martech=off

### CODE REVIEW

File: express/blocks/blog-posts/blog-posts.css [hunk 1]
```diff
@@ -0,0 +1,278 @@
+.blog main div.section:has(.blog-posts) {
+    padding: 0;
+    margin-top: 64px;
```
Comment (vhargrave) [inline express/blocks/blog-posts/blog-posts.css:3]: please don't add margin or anything to the sections.

File: express/blocks/blog-posts/blog-posts.css [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+main .section:has(.blog-posts) > div {
+    max-width: 375px;
+    margin: auto;
+    padding: 0;
+
+}
+
+.blog main .section:has(.blog-posts) > div > h2, main .section:has(.blog-posts) div.blog-posts-decoration {
+    max-width: 280px;
```
Comment (vhargrave) [inline express/blocks/blog-posts/blog-posts.css:14]: same for this. Don't modify the section styles from within the block.

**Review by vhargrave — CHANGES_REQUESTED** [review]

File: express/blocks/blog-posts/blog-posts.css (hunk 1 above)
Comment (rl-adobe) [reply express/blocks/blog-posts/blog-posts.css:3]: I couldn't find the alternative to replace the padding of the section right now.

**Comment by vhargrave** [comment]
can we close this for now?

**Pull request closed without merging**